hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/map/MapPublisherCreateWithValueMessageTask.java

```diff
@@ -98,8 +100,8 @@ private static Set<Map.Entry<Data, Data>> fetchMapSnapshotFrom(List<Future> futu
             Object result;
             try {
                 result = future.get();
-            } catch (Throwable t) {
-                throw ExceptionUtil.rethrow(t);
+            } catch (Exception e) {
```

Comment:
Any reason this is now more specific?

Reply:
Our Sonar gatekeeper was unhappy about the changed file (even the given try block was not touched). So I tried to make Sonar happier after checking the file history.